cmd/tidb-lightning-ctl/main.go

```diff
@@ -233,3 +246,51 @@ func checkpointDump(ctx context.Context, cfg *config.Config, dumpFolder string)
 	}
 	return nil
 }
+
+func unsafeCloseEngine(ctx context.Context, importer *kv.Importer, engine string) (*kv.ClosedEngine, error) {
```

Comment:
have we two versions of engineID？

Reply:
Not yet, even with the index/data engine separation the code here still works the same.